src/constants.ts

```diff
@@ -1 +1 @@
-export const serverUrl = 'https://verseny.durerinfo.hu/api/v1';
\ No newline at end of file
+export const serverUrl = 'http://localhost:5000/api/v1'; //'https://verseny.durerinfo.hu/api/v1';
```

Comment:
Do not commit this change please.

```suggestion
export const serverUrl = 'https://verseny.durerinfo.hu/api/v1';
```

Reply:
I thought this will be used everywhere (since there were an issue to change it) I change it back

Reply:
The best solution would be to use env if given, otherwise, a setting like that.